platformLibs/build.gradle

```diff
@@ -92,7 +98,7 @@ project.rootProject.ext.platformManager.enabled.each { target ->
 
         def libTask = konanArtifacts."$libName"."$targetName"
         libTask.dependsOn df.config.depends.collect{ defFileToLibName(targetName, it) }
-        libTask.dependsOn ":${targetName}CrossDist"
+//        libTask.dependsOn ":${targetName}CrossDist"
```

Comment:
Why do we need to remove this dependency?

Reply:
Sorry, it was an irrelevant change. For some reason I can build locally without it.